scripts/README.md

```diff
@@ -0,0 +1,9 @@
+Newinstall.sh
+=============
+
+This script is meant to be used for whom has no experience with conda/anaconda/miniconda.
+
+It will provide a ready to use lsstinstall environment.
+
+Use this script only if you don'y have an existing installation of
+anaconda/miniconda in your user/machine.
```

Comment:
```suggestion
This script is meant for users with no experience with conda/anaconda/miniconda.

It will provide a ready to use lsstinstall environment.

Use this script only if you don't have an existing installation of
anaconda/miniconda available.
```